XIVComboPlugin/Configuration/CustomComboPreset.cs

```diff
@@ -185,6 +188,29 @@ public enum CustomComboPreset : long
 
         [CustomComboInfo("Arcane Circle Combo", "Replace Arcane Circle with Plentiful Harvest while you have Immortal Sacrifice.", 39)]
         ReaperArcaneFeature = 1L << 30,
+
+        //PICTOMANCER
+        [CustomComboInfo("Red to Blue combo","Replace Fire combo with Blizzard combo when Subtractive Pallet is active.",42)]
```

Comment:
I think the nomenclature here is confusing
```suggestion
        [CustomComboInfo("Additive to Subtractive Combo","Replace Additive combo with Subtractive combo when Subtractive Pallet is active.",42)]
```